worldedit-core/src/main/java/com/fastasyncworldedit/core/util/WEManager.java

```diff
@@ -32,7 +32,7 @@
      * @deprecated Use {@link #weManager()} instead.
      */
     @Deprecated(forRemoval = true, since = "2.0.0")
-    private final WEManager IMP = new WEManager();
+    public static WEManager IMP = new WEManager();
```

Comment:
That would only be a temporary solution until we remove it, no?

Reply:
Yes, I will use the same example as the TaskManager to _fix_ it durably.